src/resources/views/crud/fields/repeatable.blade.php

```diff
@@ -250,7 +252,7 @@ function setupRepeatableDeleteButtonEvent(repeatable_group) {
             delete_button.click(function(){
 
                 let $repeatableElement = $(this).closest('.repeatable-element');
-                let container = $('[data-repeatable-holder='+$($repeatableElement).attr('data-repeatable-identifier')+']')
+                let container = $('[data-repeatable-holder="'+$($repeatableElement).attr('data-repeatable-identifier')+'"]')
```

Comment:
All these tiny changes for quotes and stuff? The renaming of the variables from snake_case to `camelCase`? **Next time, please do them as a separate PR or a separate commit.** I understand if doing a PR is too time-consuming, but a separate commit? Come on, that's easy 😀  

Let me explain WHY that is useful when we're working in teams, on big PRs. And why that's the way I want us to work together going forward.

If these small changes were a separate commit, I could have [cherry-picked](https://mattstauffer.com/blog/how-to-merge-only-specific-commits-from-a-pull-request/) these small changes, merge them into 4.2. They're inconsequential, I could have just gotten them out of the way, and then we would be left with the "heavy stuff" in this PR, to work on it and comment the hell out of it for days. But no, we have both small (that would get merged without even testing) and big stuff (that will definitely need a lot of back-and-forth, refactoring, etc), all in one big PR. No bueno 😀  In those cases, what you want to do is separate them... but we can't do that either, because look at the git history:

![Screenshot 2022-01-25 at 11 37 56](https://user-images.githubusercontent.com/1032474/150951560-4c2e1d5e-893b-4b3e-bda2-2ed7a7ccbf75.png)

What could I cherry-pick from these? 😅

So let's make a few new rules, to force us to be better collaborators:
- if it's a _tiny change_ (the reviewer will merge without testing), create a _separate commit_ for it;
- if it's a _small change_ (that the reviewer can test and merge separately), create a _separate PR_ for it;
- if it's a _medium change_ ... I don't know, we'll see;
- if it's a _big change_ (will definitely be some back-and-forth on it), then break it down into tiny/small/medium changes; 
    - create an issue for the big change, breaking it down into separate smaller problems that need to be tackled;
    - if there is any "low hanging fruit" that will be inconsequential and can be submitted as small PRs, do that first;
    - decide what problem should be tackled first (usually the one the others depend on);
    - create a PR for that first problem; submit the PR and wait for review; 
    - after review or green light, create a PR for the second one;
    - so on and so forth;

What's an easy way to do that? **Let's write good commit messages**. From now on:
- "refactor" is not acceptable as a commit message;
- "wip" is not acceptable as a commit message;

Every commit has to explain itself. To make it clear WHAT that commit does. Even better, **let's try never to use "_and_" in commit messages**. That will further force us to make small commits, that can easily be cherry-picked. Let's start paying attention to our git history, I think it will greatly improve our collaboration.

Another rule: **If it takes more than one day to do, it’s not a task. It’s a collection of tasks.** Treat it as such. Break it down into smaller pieces. Create an issue for it, then treat the first problem alone. Get feedback on it, then progress. See “big change” above. Then you'll never be forced to write a commit message like "wip", because tasks are manageable. Personally, I do more than that - I consider a task something that I can do in less than 3h. Something like that. If it'll take more than 3h to do... it's definitely a collection of tasks.

Sorry for bothering you with this now @pxpm , but this year I've made it a mission to improve a few things in how we collaborate. These small things will add up to a HUGE difference, I promise. And you will _love_ the results! We'll talk more in a meeting. But I'm excited about this too - I think it'll be super-good - for you, me and Backpack!

Until then, rest assured that if I took the time to write this out (especially today), it's worth taking seriously, and doing that in the future. Feel free to argue with me if you don't agree with any of the rules. But I do want to create a series of rules we stick by, from now on. Even create a document for them. Even post them on twitter or smth.

Reply:
For now, nothing to do about the above, other than read and "commit" (😅 ) to doing it. It's already too late for this PR.